### ISSUE TITLE
shrink_osd playbook fails lvm validation in stable-3.2

### ISSUE THREAD

**Issue by noahdesu** [ceph/ceph-ansible#3538]
On a fresh checkout of `stable-3.2`, the test case `tox -e mimic-shrink_osd` fails lvm_volumes validation:

```
TASK [ceph-validate : validate provided configuration] ***********************************************************************************************************************************************************************************************************************************
task path: /home/nwatkins/src/ca2/roles/ceph-validate/tasks/main.yml:2
Friday 25 January 2019  11:31:07 -0800 (0:00:00.222)       0:00:11.673 ******** 
ok: [ceph-mon0] => {
    "changed": false
}
 [ERROR]: [ceph-osd0] Validation failed for variable: lvm_volumes

 [ERROR]: [ceph-osd1] Validation failed for variable: lvm_volumes

An exception occurred during task execution. To see the full traceback, use -vvv. The error was: AttributeError: 'Invalid' object has no attribute 'message'
fatal: [ceph-osd0]: FAILED! => {}

MSG:

Unexpected failure during module execution.

An exception occurred during task execution. To see the full traceback, use -vvv. The error was: AttributeError: 'Invalid' object has no attribute 'message'
fatal: [ceph-osd1]: FAILED! => {}

MSG:

Unexpected failure during module execution.

 [ERROR]: [ceph-osd2] Validation failed for variable: lvm_volumes

An exception occurred during task execution. To see the full traceback, use -vvv. The error was: AttributeError: 'Invalid' object has no attribute 'message'
fatal: [ceph-osd2]: FAILED! => {}

MSG:

Unexpected failure during module execution.
```

**Comment by noahdesu** [comment]
@leseb the fix for this is in https://github.com/ceph/ceph-ansible/pull/3280 but it looks like you closed the backport of that to stable-3.2 https://github.com/ceph/ceph-ansible/pull/3371. Do you recall what happened?

**Comment by leseb** [comment]
@noahdesu I guess we were in a rush, after feature freeze or couldn't make it for z1 since it's kinda a feature. I guess it's fine to resurrect this, feel free to re-open the PR please. Thanks.

**Comment by noahdesu** [comment]
seems to have been fixed in stable-3.2 with test infra cleanup merged recnetly
